src/dvsim/sim_utils.py

```diff
@@ -102,17 +112,23 @@ def vcs_cov_summary_table(buf):
 
 
 def get_job_runtime(log_text: list, tool: str) -> tuple[float, str]:
-    """Returns the job runtime (wall clock time) along with its units.
+    """Return the job runtime (wall clock time) along with its units.
 
     EDA tools indicate how long the job ran in terms of CPU time in the log
     file. This method invokes the tool specific method which parses the log
     text and returns the runtime as a floating point value followed by its
     units as a tuple.
 
-    `log_text` is the job's log file contents as a list of lines.
-    `tool` is the EDA tool used to run the job.
-    Returns the runtime, units as a tuple.
-    Raises NotImplementedError exception if the EDA tool is not supported.
+    Args:
+        log_text: is the job's log file contents as a list of lines.
+        tool: is the EDA tool used to run the job.
+
+    Returns:
+        the runtime, units as a tuple.
```

Comment:
This tripped me up slightly. How about: "a tuple of (runtime, units)." ?

I originally understood it to be talking about "units as a tuple" which isn't quite right :-)

Reply:
Yea, I'm finding a lot of the docstrings here confusing. Often having to resort to reading the code.